### ISSUE TITLE
complete notifications

### ISSUE THREAD

**Issue by breadguyyy** [juandjara/wafrn-rn#6]
add notifications through unifiedpush preferably

**Comment by gabboman** [comment]
Unified push does not work on ios. Push notifications can be annoying to do. I will check because Im pretty sure some work will be required for @juandjara 's react native app and it may be reusable for the web view somehow

EDIT wrong repo lol. I will have to prepare something on the backend of wafrn for this

**Comment by juandjara** [comment]
Yeah, real notificacions are on the roadmap, but will require many things 

First registering and getting keys in the notificacions providers

Second, adapt the backend to make calls to Expo when a new notification is created (I have an example of this, written in PHP that dansup the guy from Pixelfed provided me)

And third, configure all this together in the app

It's a big feature but it will come

**Comment by juandjara** [comment]
We finally have native notifications !!
